source/2019-09-20-the-ember-times-issue-116.md

```diff
@@ -1,24 +1,24 @@
 ---
 title: The Ember Times - Issue No. 116
-author: the crowd
+author: Chris Ng, the crowd
 tags: Recent Posts, Newsletter, Ember.js Times, Ember Times, 2019
 alias : "blog/2019/09/20-the-ember-times-issue-116.html"
 responsive: true
 ---
 
 <SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹
-
+Check out the new Octane Super Rentals Tutorial 🚀,
 <SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## [Octane Super Rentals Tutorial 🚀](https://octane-guides-preview.emberjs.com/release/tutorial/01-orientation/)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-  
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+[@chancancode](https://github.com/chancancode) and [@vaidehijoshi](https://github.com/vaidehijoshi) did some great work writing content and creating a tutorial code generator that runs commands, inserts the code into a tutorial, and even takes screenshots. You can now preview the [Octane guides](https://octane-guides-preview.emberjs.com/release/tutorial/01-orientation/) and check out the tutorial.
```

Comment:
```suggestion
[Godfrey Chan @chancancode](https://github.com/chancancode) and [Vaidehi Joshi @vaidehijoshi](https://github.com/vaidehijoshi) did some great work writing content and creating a tutorial code generator that runs commands, inserts the code into a tutorial, and even takes screenshots. You can now preview the [Octane guides](https://octane-guides-preview.emberjs.com/release/tutorial/01-orientation/) and check out the tutorial.
```

Reply:
I thought our conventions was to use the at github alias?

Reply:
We've kinda been doing both lately e.g. https://blog.emberjs.com/2019/09/13/the-ember-times-issue-115.html. I guess someone did it and I thought it might be a bit more friendly

Reply:
What do you think? Def don't have to keep going this way

Reply:
Minor suggestion: maybe take out `and creating a tutorial code generator that runs commands, inserts the code into a tutorial, and even takes screenshots` in this first paragraph and combine it into the second one, so that the first paragraph is focused on content and the second one is focused on infrastructure.

Reply:
Updated the text based on @chancancode 's comment and ill accept this (@amyrlam 's change) for the non-alias name

Reply:
Oh I guess I can't accept it now so i'll push in a  new commit :)